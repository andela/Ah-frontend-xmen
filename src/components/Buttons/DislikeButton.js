import React from 'react';

const DislikeButton = (props) => {
  const classNames = props.disliked ? 'btn btn-sm btn-default' : 'btn btn-sm btn-outline-default';
  return (
    <button type="button" id="like-btn" onClick={props.handleDislike} className={classNames}>
      <span className="pr-1"><i className="fa fa-sm fa-thumbs-down" /></span>
      {props.dislikesCount}
    </button>
  );
};

export default DislikeButton;
