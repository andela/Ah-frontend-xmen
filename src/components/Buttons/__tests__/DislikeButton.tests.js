import React from 'react';
import { shallow } from 'enzyme';
import DislikeButton from '../DislikeButton';

describe('DislikeButton', () => {
  it('should render without crashing', () => {
    const wrapper = shallow(<DislikeButton />);
    expect(wrapper).toMatchSnapshot();
  });
});
