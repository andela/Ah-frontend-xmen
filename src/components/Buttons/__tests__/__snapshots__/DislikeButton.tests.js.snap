// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DislikeButton should render without crashing 1`] = `ShallowWrapper {}`;
